src/module_library/c3photo.cpp

```diff
@@ -150,22 +150,17 @@ photosynthesis_outputs c3photoC(
     });                                        // micromol / m^2 / s
 
     // Run the secant method
-    double Assim_check{};  // Will be modified by find_root_secant_method
-    size_t iterations;     // Will be modified by find_root_secant_method
-
-    double const co2_assim_rate = find_root_secant_method(
+    // double Assim_check{};  // Will be modified by find_root_secant_method
+    // size_t iterations;     // Will be modified by find_root_secant_method
```

Comment:
Delete these lines instead of commenting them out, since they aren't used anymore.